language/en/units.json

```diff
@@ -900,14 +900,14 @@
 			"armcom": "Commander",
 			"armcomboss": "Oh s##t we are so dead",
 			"armcomlvl2": "Commander",
-			"armcomlvl3": "Specialized in defensive offenses",
-			"armcomlvl4": "Specialized in defensive offenses",
-			"armcomlvl5": "Specialized in defensive offenses",
-			"armcomlvl6": "Specialized in defensive offenses",
-			"armcomlvl7": "Specialized in defensive offenses",
-			"armcomlvl8": "Specialized in defensive offenses",
-			"armcomlvl9": "Specialized in defensive offenses",
-			"armcomlvl10": "Specialized in defensive offenses",
+			"armcomlvl3": "Defensive specialist that shares experience with turrets",
```

Comment:
Please use unit def custom param `i18nfromunit` to avoid repeated entries.
[See Proxy Entries section of dev translantion guide](https://docs.google.com/document/d/1vLREk5nC8POWHngipz4pMnQsi1qFw_8pyUaEX6Gwo-I/edit#heading=h.6ji1mejd12ju)

Reply:
Please let this quick fix through and I will do as your request over the next few weeks for all of the evolving Commanders in a separate pull request.